frontend/src/modules/organization/components/organization-manage-domains-drawer.vue

```diff
@@ -84,12 +96,30 @@ const drawerModel = computed({
   },
 });
 
-const domainsIdentities = ref<OrganizationIdentity[]>(
-  [...(props.organization.identities.filter((i) => [
-    OrganizationIdentityType.ALTERNATIVE_DOMAIN,
-    OrganizationIdentityType.PRIMARY_DOMAIN,
-  ].includes(i.type)) || [])],
-);
+const platformLabel = (platforms: string[]) => CrowdIntegrations.getPlatformsLabel(platforms);
+
+const domainsIdentities = ref(domains(props.organization) || []);
```

Comment:
Should this be a computed property? When organization is fetched, will domainIdentities be updated?

Reply:
No because the value is being set to this property